metadata.md

```diff
@@ -24,7 +24,7 @@ Comment on the ERD in [Lucidcharts](https://lucid.app/lucidchart/invitations/acc
 
 ## Sample
 
-The sample is a representative volume of wastewater taken from a site which is then analysed by a lab.
```

Comment:
@DougManuel or @NielsNicolai Why was 'representative' removed? Unless the sample is representative of the wastewater or sludge stream it's then fundamentally flawed and we can't rely on the results. Unless I'm misunderstanding something. Quickly read through the rest and did not notice any glaring issues.

Reply:
@DougManuel  and @NielsNicolai  meant to say in the last statement  in the last statement of the above comment  that I read the whole metadata.md file and did not notice any glaring issues.

Reply:
@NielsNicolai Sorry, I didn't finish edits to the the description of tables. Can you review again? We can put back 'representative'. I didn't feel that this descriptive is important for metadata since we are not commenting on methods in other sections.

Reply:
@NielsNicolai and @DougManuel sure, but this may be the exception where I would put 'representative' back in.

Reply:
@vipileggi The word representative is back in the description :+1: